client/src/components/SubmittedDesigns/SubmittedDesigns.tsx

```diff
@@ -0,0 +1,63 @@
+import useStyles from './useStyles';
+
+import ImageList from '@material-ui/core/ImageList';
+import ImageListItem from '@material-ui/core/ImageListItem';
+import ImageListItemBar from '@material-ui/core/ImageListItemBar';
+
+{
+  /* these images are temporary, will be deleted, After getting images as props from Contest.tsx file*/
+}
+import image1 from '../../Images/48bbc97ff2ad97160445538959a224e642ce5816.png';
+import image2 from '../../Images/612bd8560dbfd2834c5d539bf0a1055d505f48a4.png';
+import image3 from '../../Images/612bd8560dbfd2834c5d539bf0a1055d505f48a4.png';
+import image4 from '../../Images/612bd8560dbfd2834c5d539bf0a1055d505f48a4.png';
+
+interface Props {
+  images: [];
+  artist: string;
+}
+
+{
+  /* tempImages is only for testing purpose, have to delete them later, after getting images from api*/
+}
+const tempImages = [
+  {
+    src: image1,
+    artist: 'jesse',
+  },
+  {
+    src: image2,
+    artist: 'anthony',
+  },
+  {
+    src: image3,
+    artist: 'denise',
+  },
+  {
+    src: image4,
+    artist: 'james',
+  },
+];
+
+const SubmittedDesigns = ({ images, artist }: Props): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <div style={{ textAlign: 'center' }}>
```

Comment:
You can use an MUI Box here instead, it has flexbox built in as well

Reply:
made changes using MUI Box ,